bridge/bridge.go

```diff
@@ -362,9 +362,27 @@ func (b *Bridge) remove(containerId string, deregister bool) {
 				log.Println("removed:", containerId[:12], service.ID)
 			}
 		}
+		cleanUpAttrs := func(services []*Service) {
+		    b.registry.AcquireDistributedLock()
+            distServices, err := b.registry.DistributedServices()
```

Comment:
@ericjohnlucas I think this PR has the wrong branch?  We fixed this on https://github.com/ericjohnlucas/registrator/tree/master but this PR is for https://github.com/ericjohnlucas/registrator/tree/feature/saveAttrsInKV